cartoframes/auth/__init__.py

```diff
@@ -6,7 +6,7 @@
 _default_context = None
 
 
-def set_default_context(context=None, base_url=None, api_key=None, creds=None, session=None):
+def set_default_context(context=None, base_url=None, api_key='default_public', creds=None, session=None):
```

Comment:
I'm missing the `Args` section of the docs here. Is that going to happen package-wide in another PR?